### PR TITLE
Clarified some basics

### PR DESCRIPTION
**Pull request by kmlawson**
Just adding a bit of basic information about the fact the user is to create a file to enter this example material into.

### CODE REVIEW

File: doc/intro.rst [hunk 1]
```diff
@@ -7,6 +7,8 @@ adventurelib.
 Starting a project
 ------------------
 
+Your text adventure will be run from a single python file. Create a new plain text file (we'll call it my_game.py) and open it in your favorite editor.
```
Comment (lordmauve) [inline doc/intro.rst]: Adventurelib doesn't require that games run from a single file, so perhaps you could clarify that this applies for this example.

File: doc/intro.rst [hunk 2]
```diff
@@ -49,14 +51,14 @@ All of the rest of your code should go in between the ``from adventurelib
 import *`` and the ``start()`` lines.
 
 We can use the :doc:`@when <commands>` syntax to create a command that player
-can type in order to interact with your game. Let's add a ``brush teeth``
+can type in order to interact with your game. Opening your file once again, let's add a ``brush teeth``
```
Comment (lordmauve) [inline doc/intro.rst]: Lots of teachers use IDLE to teach with, and all this closing and opening isn't necessary. I'd prefer to leave out discussion about how to use the editor.

Comment (kmlawson) [reply doc/intro.rst]: Ah, I see. Hadn't thought of that. I have never programmed with Python before so was going into your code blind and stumbled in places. Thanks for all your hard work on this. Looks like a great way to teach some Python to youth. In my case I just wanted to throw together a text editor for a nephew. I ended up switching to a different library here on github called bwx-adventure, which though also for teaching, didn't force you to build so much from scratch. Anyways, thanks again and I hope the project continues to develop!

Comment (lordmauve) [reply doc/intro.rst]: That's fine; the "from scratch" approach is intentional and might not be what everyone needs.

I'll close this PR. Thank you for your docs review!

**Pull request closed without merging**